### PR TITLE
Fix PE UG bugs

### PR DESCRIPTION
**Pull request by RyanCheungJF**
Resolves https://github.com/AY2122S2-CS2103T-T09-2/tp/issues/209

Resolves https://github.com/AY2122S2-CS2103T-T09-2/tp/issues/205 

Resolves https://github.com/AY2122S2-CS2103T-T09-2/tp/issues/187

Resolves https://github.com/AY2122S2-CS2103T-T09-2/tp/issues/182

Resolves https://github.com/AY2122S2-CS2103T-T09-2/tp/issues/174

Resolves https://github.com/AY2122S2-CS2103T-T09-2/tp/issues/171

Resolves https://github.com/AY2122S2-CS2103T-T09-2/tp/issues/166

https://ryancheungjf.github.io/tp/UserGuide.html

#226 
NOTE: I've added a section on constraints of each command, would be great if each developer could update it when they work on UG as well!

Also, under the Mac section of Quick Start, we have to ensure that the release matches the documentation.

The main thing to check would be the target user group at the very start. LMK if it is not comprehensive.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -58,8 +71,8 @@ Cautions are placed in this guide to serve as warnings for certain actions.
   * **`delete`**`INDEX` : Deletes the applicant from the list of applicants.
 
   * **`exit`** : Exits the app.
```
Comment (zzhikai) [inline docs/UserGuide.md:73]: Indent these bullet points as mentioned in #166, use tab can already inside the markdown file

**Review by zzhikai — COMMENTED** [review]
I don't think #205, #171 has been resolved, see comment on #166 as well

File: docs/UserGuide.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ title: User Guide
 
 LinkedOUT is the only application that any experienced recruiter needs. LinkedOUT allows recruiters to keep track of many applicants, and the job they applied for. You can store their contact details, skills and the round of their application, all in one place.
 
+LinkedOUT helps recruiters manage the multiple job applications they may receive on a daily basis. With many applications, it may be difficult to keep track of each applicant and which application round they are currently at.
+
+Thus, LinkedOUT aims to improve a recruiter's experience. LinkedOUT presents recruiters with the ability to flag important applicants, edit applicants easily and search for them with ease.
+
+LinkedOUT comes with a Command Line Interface ([CLI](#glossary)) as well as a Graphical User Interface ([GUI](#glossary)) in order to provide recruiters a more streamlined experience.
```
Comment (zzhikai) [inline docs/UserGuide.md:12]: Is this meant to be under the section: Who is this User Guide for  ? #171

File: docs/UserGuide.md (hunk 1 above)
Comment (RyanCheungJF) [reply docs/UserGuide.md:73]: Thanks, fixed!

File: docs/UserGuide.md (hunk 2 above)
Comment (RyanCheungJF) [reply docs/UserGuide.md:19]: I've moved it after our discussion, do help me check again. Thanks!

**Review by zzhikai — APPROVED** [review]
Great work LGTM!

**Pull request merged**